config/settings.py

```diff
@@ -123,3 +124,63 @@
 # https://docs.djangoproject.com/en/5.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+REST_FRAMEWORK = {
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
+    'PAGE_SIZE': 6 ,
+}
+
+LOG_DIR = os.path.join(BASE_DIR, 'logs')
+
+if not os.path.exists(LOG_DIR):
+    os.makedirs(LOG_DIR)
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+        },
+        'http_file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': os.path.join(LOG_DIR, 'http_logs.log'),  # Путь к файлу логов
+        },
+        'db_file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': os.path.join(LOG_DIR, 'db_logs.log'),
+        }
+    },
+    'loggers': {
+        'django': {
```

Comment:
UPD: Заметил, что все-таки логирования HTTP запросов не происходит. То что пишется в файл http_logs это работа родительского логгера, но там нет HTTP. Логгер django.request логгирует только исключеня уровня ERROR и выше. Нужно чуть исправить
```suggestion
        'django.server': {
```